src/main/java/modfest/teamgreen/mixin/AccessorAxeItem.java

```diff
@@ -0,0 +1,25 @@
+package modfest.teamgreen.mixin;
+
+import java.util.Map;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.AxeItem;
+
+@Mixin(AxeItem.class)
+public interface AccessorAxeItem {
```

Comment:
fabric api should have this

Reply:
FabricMC/fabric#576 exists, but it hasn't been merged yet.